lighthouse-core/audits/accessibility/empty-table-header.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Ensures table headers have discernible text.
+ * See base class in axe-audit.js for audit() implementation.
+ */
+
+const AxeAudit = require('./axe-audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of an accesibility audit that checks if there are any table headers without readable text. This title is descriptive of the successful state and is shown to users when no user action is required. */
+  title: 'Table headers have discernible text',
```

Comment:
do we want to do ``'`<table>` headers have discernible text'`` here to keep it from being translated?

Reply:
the axe docs also just use "visible text". Seems more straightforward than "discernible"?